### ISSUE TITLE
status code does not indicate success: 404 (Feed does not exist.)

### ISSUE THREAD

**Issue by s0dyy** [dotnet/source-build#1971]
Hello ;)

I have a 404 since yesterday from ./build.sh, it was working fine a few days ago.

Is there a problem or change with https://dotnet.myget.org/F/nuget-build/api/v3/index.json ?

```
dotnet-install:` Installation finished successfully.
/home/s0dyy/source-build/.dotnet/sdk/5.0.100/MSBuild.dll /nologo -maxcpucount /m -verbosity:m /v:minimal /bl:/home/s0dyy/source-build/artifacts/log/Debug/Build_0112095617.binlog /clp:Summary /flp:v=diag /nr:true /p:TreatWarningsAsErrors=false /p:ContinuousIntegrationBuild=false /p:Configuration=Release /p:RepoRoot=/home/s0dyy/source-build /p:Restore=true /p:Build=true /p:Rebuild=false /p:Test=false /p:Pack=false /p:IntegrationTest=false /p:PerformanceTest=false /p:Sign=false /p:Publish=false /home/s0dyy/source-build/packages/restored/microsoft.dotnet.arcade.sdk/5.0.0-beta.20426.4/tools/Build.proj
/home/s0dyy/source-build/packages/restored/microsoft.dotnet.arcade.sdk/5.0.0-beta.20426.4/tools/ProjectLayout.props(8,3): warning MSB4011: "/home/s0dyy/source-build/packages/restored/microsoft.dotnet.arcade.sdk/5.0.0-beta.20426.4/tools/RepoLayout.props" cannot be imported again. It was already imported at "/home/s0dyy/source-build/packages/restored/microsoft.dotnet.arcade.sdk/5.0.0-beta.20426.4/tools/Build.proj (49,3)". This is most likely a build authoring error. This subsequent import will be ignored.
  Determining projects to restore...
/home/s0dyy/source-build/.dotnet/sdk/5.0.100/NuGet.targets(131,5): error : Unable to load the service index for source https://dotnet.myget.org/F/nuget-build/api/v3/index.json. [/home/s0dyy/source-build/packages/restored/microsoft.dotnet.arcade.sdk/5.0.0-beta.20426.4/tools/Tools.proj]
/home/s0dyy/source-build/.dotnet/sdk/5.0.100/NuGet.targets(131,5): error :   Response status code does not indicate success: 404 (Feed does not exist.). [/home/s0dyy/source-build/packages/restored/microsoft.dotnet.arcade.sdk/5.0.0-beta.20426.4/tools/Tools.proj]

Build FAILED.

/home/s0dyy/source-build/packages/restored/microsoft.dotnet.arcade.sdk/5.0.0-beta.20426.4/tools/ProjectLayout.props(8,3): warning MSB4011: "/home/s0dyy/source-build/packages/restored/microsoft.dotnet.arcade.sdk/5.0.0-beta.20426.4/tools/RepoLayout.props" cannot be imported again. It was already imported at "/home/s0dyy/source-build/packages/restored/microsoft.dotnet.arcade.sdk/5.0.0-beta.20426.4/tools/Build.proj (49,3)". This is most likely a build authoring error. This subsequent import will be ignored.
/home/s0dyy/source-build/.dotnet/sdk/5.0.100/NuGet.targets(131,5): error : Unable to load the service index for source https://dotnet.myget.org/F/nuget-build/api/v3/index.json. [/home/s0dyy/source-build/packages/restored/microsoft.dotnet.arcade.sdk/5.0.0-beta.20426.4/tools/Tools.proj]
/home/s0dyy/source-build/.dotnet/sdk/5.0.100/NuGet.targets(131,5): error :   Response status code does not indicate success: 404 (Feed does not exist.). [/home/s0dyy/source-build/packages/restored/microsoft.dotnet.arcade.sdk/5.0.0-beta.20426.4/tools/Tools.proj]
    1 Warning(s)
    1 Error(s)
```

I think there is a similar issue on aspnetcore and powershell:

https://github.com/dotnet/aspnetcore/issues/29207
https://github.com/PowerShell/PowerShell/issues/14598

**Comment by dseefeld** [comment]
Yes. myget feeds have been deprecated and source-build has not been updated, yet.  A fix is in progress.  I'll update this issue when a fix is ready.

**Comment by dseefeld** [comment]
This issue is fixed for master #1972 and release/3.1 #1974
